src/page_body/structure/blocks/page_block_figma_components.pr

```diff
@@ -1 +1,104 @@
-Test 1
\ No newline at end of file
+{[ const block = context /]}
+
+{[ let configuration = exportConfiguration() /]}
+
+{[ const brands = ds.allBrands() /]}
+{[ const components = ds.allDesignComponents(brands[0].id) /]}
+
+{* TODO: this is not propagated from BE *}
+{[ let numberOfColumns = 2 /]}
+{[ let variant = "Grid" /]}
+
+{[ if (variant === "Grid") ]}
+    <div class="figma-components" style="grid-template-columns: repeat({{ numberOfColumns }}, 1fr);">
+        {[ for component in components ]}
+            {[ if (isComponentSelected(block.selectedFigmaComponents, component.id)) ]}
+                {[ const selectedPropertiesIds = block.selectedFigmaComponents[0].selectedComponentProperties /]}
+                {[ const componentPreviews = getComponentPreviews(component, selectedPropertiesIds, block.selectedFigmaComponents.length > 1) /]}
+                {[ for componentPreview in componentPreviews ]}
+                    <div class="figma-component">
+                        <div class="figma-component-image figma-component-image-border figma-component-image-background" style="{{ block.figmaComponentsBlockConfig.previewContainerSize === "Centered" ? "height: 116px;" : "" }}">
+                            <img src="{{ componentPreview.component.thumbnailUrl }}" alt="{{ componentPreview.name }}" height="{{ componentPreview.component.origin?.height }}" width="{{ componentPreview.component.origin?.width }}" />
+                        </div>
+                        <div class="figma-component-details">
+                            {[ if (block.figmaComponentsBlockConfig.showComponentName) ]}
```

Comment:
Add name to the if check

Reply:
[2b424fa](https://github.com/Supernova-Studio/exporter-documentation/pull/167/commits/2b424fa39b2902792b580764ef3661118bd8d347)